resources/js/src/app/components/itemList/filter/ItemDropdown.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <div class="fl-dropdown">
     <div
-      v-for="value in facet.values.slice(0, facet.itemCount)"
+      v-if="facet.values[0].frequency"
+      v-for="value in facet.values"
```

Comment:
1. v-for and v-if should not be used on the same component. If needed, please wrap it in a template tag
2. What's the reason for this check? The values without frequency shouldn't be returned anyway, I think.